src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -106,7 +106,15 @@ static bool templateNotify
   if (httpInfo.payload == "")
   {
     LM_W(("KZ: using default payload"));
-    payload = "{" + ce.toJson(renderFormat, attrsOrder) + "}";
+    NotifyContextRequest ncr;
+    ContextElementResponse cer;
+    cer.contextElement = ce;
+    ncr.subscriptionId = subscriptionId;
+    ncr.contextElementResponseVector.push_back(&cer);
+    payload = ncr.toJson(renderFormat, attrsOrder);
+
+    //payload = "{" + ce.toJson(renderFormat, attrsOrder) + "}";
```

Comment:
If  you want to keep this comment, please add a space after // (the current style-guide checker complains about this).
And more style-guide: please align ncr and cer where declared.